static/include/config.hpp

```diff
@@ -0,0 +1,13 @@
+#ifndef PYARBORX_CONFIG_H
+#define PYARBORX_CONFIG_H
+
+#include <ArborX_Point.hpp>
+#include <ArborX_Box.hpp>
+#include <ArborX_Sphere.hpp>
+
+#include <pybind11/stl.h>
+#include <pybind11/stl_bind.h>
+
+PYBIND11_MAKE_OPAQUE(std::vector<ArborX::Point>);
+PYBIND11_MAKE_OPAQUE(std::vector<ArborX::Box>);
```

Comment:
These create automatic wrappers for std::vector with the corresponding `ArborX` types. This is not so much for our wrapper but allows another library to have std::vector of ArborX types in their interfaces (that they want to call from python). As the static part of the bindings is only compiled once the extra workload to compile is only paid once.

But we also can add these later

Reply:
I have mixed feelings about these ones.  That's for codes that would write C++ and include ArborX right?

Reply:
I removed these to make the pr cleaner ... we can still add them later